itou/users/management/commands/import_ai_employees.py

```diff
@@ -0,0 +1,335 @@
+CONTRACT_STARTDATE_COL = "ctr_date_embauche"
+CONTRACT_ENDDATE_COL = "ctr_date_fin_reelle"
+COUNTRY_COL = "adr_code_insee_pays"
+EMAIL_COL = "adr_mail"
+FIRST_NAME_COL = "pph_prenom"
+GENDER_COL = "pph_sexe"
+LAST_NAME_COL = "pph_nom_usage"
+NIR_COL = "ppn_numero_inscription"
+PHONE_COL = "adr_telephone"
+POST_CODE_COL = "codepostalcedex"
+SIAE_NAME_COL = "pmo_denom_soc"
+SIRET_COL = "pmo_siret"
+
+DATE_FORMAT = "%Y-%m-%d"
+
+
+class Command(BaseCommand):
+    """ """
+
+    help = "Import AI employees and deliver a PASS IAE."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry-run", dest="dry_run", action="store_true", help="Don't change anything in the database."
+        )
+        parser.add_argument(
+            "--sample-size",
+            dest="sample_size",
+            help="Sample size to run this script with (instead of the whole file).",
+        )
+        parser.add_argument(
+            "--file-path",
+            dest="file_path",
+            required=True,
+            action="store",
+            help="Absolute path of the CSV file to import",
+        )
+        parser.add_argument(
+            "--email",
+            dest="developer_email",
+            required=True,
+            action="store",
+            help="Developer email account (in the Itou database).",
+        )
+
+    def set_logger(self, verbosity):
+        """
+        Set logger level based on the verbosity option.
+        """
+        handler = logging.StreamHandler(self.stdout)
+
+        self.logger = logging.getLogger(__name__)
+        self.logger.propagate = False
+        self.logger.addHandler(handler)
+
+        self.logger.setLevel(logging.INFO)
+        if verbosity >= 1:
+            self.logger.setLevel(logging.DEBUG)
+
+    def get_ratio(self, first_value, second_value):
+        return round((first_value / second_value) * 100, 2)
+
+    def fix_dates(self, date):
+        # This is quick and ugly!
+        if date.startswith("16"):
+            return date[0] + "9" + date[2:]
+        return date
+
+    def log_to_csv(self, csv_name, logs):
+        csv_file = Path(settings.EXPORT_DIR) / f"{csv_name}.csv"
+        with open(csv_file, "w", newline="") as file:
+            if isinstance(logs, list):
+                fieldnames = list(logs[0].keys())
+                writer = csv.DictWriter(file, fieldnames=fieldnames)
+                writer.writeheader()
+                writer.writerows(logs)
+            else:
+                file.write(logs.to_csv())
+
+    def clean_nir(self, row):
+        nir = row[NIR_COL]
+        try:
+            validate_nir(nir)
+        except ValidationError:
+            return None
+        return nir
+
+    def fake_email(self, first_name, last_name):
+        random_number = random.randrange(100, 10000)
+        first_name = slugify(first_name)
+        last_name = slugify(last_name)
+        return f"{first_name}-{last_name}-{random_number}@faux-email.com"
```

Comment:
On pas le choix 😢

Juste pour la forme je propose ça, qui te fait économiser quelques lignes et un import :

```suggestion
        return f"{uuid.uuid4().hex}@fake-email.com"
```

Reply:
Ou si un autre `save()` ne te fais pas peur lors de la création de l'utilisateur, tu as aussi `jobseeker_hash_id` (il faut avoir la PK)

Reply:
Je prends la proposition. Merci !